fieldservice_account/models/fsm_order.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (C) 2018 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields
+
+from odoo.addons.base_geoengine import geo_model
+
+
+class FSMOrder(geo_model.GeoModel):
+    _inherit = 'fsm.order'
+
+    contractor_cost_ids = fields.One2many('account.invoice.line',
+                                          'fsm_order_id',
+                                          string='Contractor Costs')
+    employee_timesheet_ids = fields.One2many('account.analytic.line',
+                                             "fsm_order_id",
+                                             string='Employee Timesheets')
+    total_cost = fields.Float(compute='_compute_total_cost',
+                              string='Total Cost')
+
+    employee = fields.Boolean(compute='_compute_employee')
+
+    def _compute_employee(self):
+        user = self.env['res.users'].browse(self.env.uid)
+        for order in self:
+            if user.employee_ids:
+                order.employee = True
+
+    @api.depends('employee_timesheet_ids', 'contractor_cost_ids')
+    def _compute_total_cost(self):
+        for order in self:
+            order.total_cost = 0.0
+            for line in order.employee_timesheet_ids:
+                for emp in line.user_id.employee_ids:
+                    rate = emp.timesheet_cost
+                    continue
+                order.total_cost += line.unit_amount * rate
+            for cost in order.contractor_cost_ids:
+                order.total_cost += cost.price_unit * cost.quantity
+
+    def action_complete(self):
+        for order in self:
+            if order.contractor_cost_ids:
+                order._create_vendor_bill()
+            if order.employee_timesheet_ids:
+                order._create_customer_invoice()
+        return super().action_complete()
```

Comment:
`return super(FSMOrder, self).action_complete()`

Reply:
Fixed in new commit